app/src/main/java/net/gsantner/opoc/util/GsFileUtils.java

```diff
@@ -752,4 +754,40 @@ public static List<File> replaceFilesWithCachedVariants(@Nullable List<File> fil
         }
         return files;
     }
+
+    // Check if a file can be created (parent exists and can be written)
```

Comment:
why not use /** ... */ instead of // which will allow Androidstudio to show tooltip help when mouse is over the method?